File: beginner/haromszog.js

let aszam=3; let bszam=4; let cszam=5;

  if (
    aszam + bszam >= cszam &&     aszam + cszam >= bszam &&     bszam + cszam >= aszam
  ) {
    console.log("szerkeszthető");
    if (aszam === bszam || aszam === cszam || bszam === cszam) {
      console.log("egyenlő szárú háromszög");}
    else if ( aszam * aszam + bszam * bszam === cszam * cszam || aszam * aszam + cszam * cszam === bszam * bszam || bszam * bszam + cszam * cszam === aszam * aszam
    ) {
      console.log("derékszögű háromszög");    }
    else {
      console.log("általános háromszög");    }
  }
  else {
    console.log("nem szerkeszthető");
  }

File: beginner/haromszog.py
aszam=3
bszam=4
cszam=5

if aszam + bszam >= cszam and aszam + cszam >= bszam and bszam + cszam >= aszam:
    print ("szerkesthető")
    if aszam == bszam or aszam==cszam or bszam==cszam:
        print ("egyenló szárú háromszög")
    elif aszam*aszam+bszam*bszam==cszam*cszam:
        print ("derékszögű háromszög")
    else:
        print ("általános háromszög")
else:
    print ("nem szerkeszthető")